lib/Tools/dedukti_test

```diff
@@ -4,8 +4,17 @@
 
 set -e
 
-OUTPUT="${1:-output.lp}"
+OUTPUT="${1:-isabelle_dedukti_test}"
 
-isabelle dedukti_import -O "$OUTPUT" Dedukti_Import
+if [ -d $OUTPUT ]; then
+    if [ ! -f "$OUTPUT/lambdapi.pkg" ]; then
+        echo "$OUTPUT is not a lambdapi package directory"
+        exit 1
+    fi
+else
+    $ISABELLE_LAMBDAPI init "${OUTPUT//"/"/.}"
```

Comment:
What does this line with the `//` do? Can you add a comment?

Reply:
My goal, and what `${OUTPUT//"/"/.}` should do, is replace every "/" in the argument by ".".
lambdapi expects a package root, which features directory nesting with "." instead of the Unix "/"